adapters/sinatra/lib/opentelemetry/adapters/sinatra/extensions/tracer_extension.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require_relative '../middlewares/tracer_middleware'
+
+module OpenTelemetry
+  module Adapters
+    module Sinatra
+      module Extensions
+        module TracerExtension
+          # Sinatra hook after extension is registered
+          def self.registered(app)
+            # Create tracing `render` method
+            ::Sinatra::Base.module_eval do
+              def render(engine, data, *)
+                Sinatra::Adapter.tracer.in_span(
+                  'sinatra.render_template',
+                  kind: :server,
```

Comment:
`kind:` shouldn't be specified here. It will default to `:internal`. This span isn't a server span - it's parent span is (the middleware span).

Reply:
fixed :heavy_check_mark: by removing `kind:`